Pegasus/sources/Math.cpp

```diff
@@ -3,7 +3,7 @@
 * This code is licensed under the MIT license (MIT)
 * (http://opensource.org/licenses/MIT)
 */
-#include "Pegasus/include/Math.hpp"
+#include <Pegasus/include/Math.hpp>
```

Comment:
It's something that should be in our code style if it is not already, that we use `<>` in both cases. And not it's a convention it's just a way to say the compiler where it should look for the file in a first place for what I know.